apps/web-giddh/src/app/settings/taxes/setting.taxes.component.ts

```
@@ -85,56 +93,67 @@ export class SettingTaxesComponent implements OnInit, OnDestroy {
-                }
-            });
-
-        this.store
-            .pipe(select(state => state.company && state.company.isTaxUpdatedSuccessfully), takeUntil(this.destroyed$))
-            .subscribe(result => {
-                if (result && this.taxAsideMenuState === 'in') {
-                    this.toggleTaxAsidePane();
-                }
-            });
+        combineLatest([
+            this.store.pipe(select(state => state.company && state.company.isTaxCreatedSuccessfully)),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)